src/paint.rs

```diff
@@ -24,6 +25,8 @@ pub struct Painter<'a> {
     pub highlighter: HighlightLines<'a>,
     pub config: &'a config::Config<'a>,
     pub output_buffer: String,
+    pub line_number_in: usize,
+    pub line_number_out: usize,
```

Comment:
To be consistent with other variables names, would it make sense for these to be named `minus_line_number` and `plus_line_number`?

Reply:
Yep.